Makefile

```diff
@@ -134,6 +136,21 @@ ocs-registry-master:
 	@echo "Building ocs-registry image in appregistry format using master images for all OCS components"
 	hack/build-master-appregistry.sh
 
+catalog-build: opm
+	@echo "Building ocs-operator-catalog"
+	${OPM} render --output=yaml ${OCS_BUNDLE_IMG} > catalog/ocs-bundle.yaml
+	${OPM} render --output=yaml ${NOOBAA_BUNDLE_IMG} > catalog/noobaa-bundle.yaml
```

Comment:
We don't need noobaa. But, we probably need rook.

Reply:
We do need noobaa to build a catalog as noobaa CSV needs to be installed too

Reply:
Yes, Nooba is needed for ocs to be installed and storagecluster to get to ready.